### PR TITLE
add QTUM, ETC, BCC/BCH

### PR DESCRIPTION
**Pull request by lucidash**
Add ETC, BCC/BCH, QTUM currencies.

### CODE REVIEW

File: lib/mtree/world.rb [hunk 1]
```diff
@@ -65,6 +72,15 @@ def add_oneway(base, target, exchanger)
 
     general = currencies.reject { |cur| cur.coin == 'KRW' }
 
+    general
```
Comment (ermaker) [inline lib/mtree/world.rb]: How about merging this and line 84?

Comment (lucidash) [reply lib/mtree/world.rb]: I.... don't know  how to do that... Sorry ,,,

**Comment by lucidash** [comment]
Yes, absolutely.

**Comment by ermaker** [comment]
Perfect.

**Pull request merged**